### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by garethmullins**
@AdamBorg, Would you mind going through my code to review it?
The reason for two requests was that I messed up the first one, sorry about that.

### CODE REVIEW

**Comment by AdamBorg** [comment]
So sorry this is done so late I actually have so much trouble navigating gitHub!!!

File: cumulativeTotals.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+def income_report(incomes, num_of_months):
+    print("\nIncome Report\n-------------")
+    total = 0
+    for month in range(1, num_of_months + 1):
+        income = incomes[month - 1]
+        total += income
+        print("Month {:2} - Income: ${:10.2f} Total: ${:10.2f}".format(month, income, total))
+
+
```
Comment (AdamBorg) [inline cumulativeTotals.py:27]: I can not see anything major so I am just being picky. More commenting to say whats is happening although it is quiet obvious.

File: listComprehensions.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    for name in names:
+        first_initials.append(name[0])
+        print(first_initials)
+
+    # list comprehension that does the same thing
+    first_initials = [name[0] for name in names]
+    print(first_initials)
+
+    # list comprehension that creates a list containing tuples of both initials
+    # splits each name and adds the first letter of each part to a tuple
```
Comment (AdamBorg) [inline listComprehensions.py]: As it is a quiet the chunk of commenting might be worth putting it in to a docstring.

File: number_inputs.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+"""
+__author__ = 'Gareth'
+
+
+def main():
+    numbers = []
+    NUM_OF_NUMBERS = 5
+    for i in range(0, NUM_OF_NUMBERS):
+        numbers.append(int(input(" please enter number {} ".format(i + 1))))
+    print("The first number is {}".format(numbers[0]))
```
Comment (AdamBorg) [inline number_inputs.py:13]: create a function for all the outputs?

**Review by AdamBorg — COMMENTED** [review]
So sorry this is done so late I actually have so much trouble navigating gitHub!!!

File: number_inputs.py (hunk 3 above)
Comment (garethmullins) [reply number_inputs.py:13]: I would need to create a function for each print, then either call each function or call another function to call each function, therefore, I don't think putting them into function(s) would be beneficial

**Pull request merged**